src/components/views/dialogs/FeedbackDialog.tsx

```diff
@@ -30,19 +30,23 @@ const existingIssuesUrl = "https://github.com/vector-im/element-web/issues" +
     "?q=is%3Aopen+is%3Aissue+sort%3Areactions-%2B1-desc";
 const newIssueUrl = "https://github.com/vector-im/element-web/issues/new/choose";
 
-export default (props) => {
-    const [rating, setRating] = useState("");
-    const [comment, setComment] = useState("");
+interface IProps {
+    onFinished: () => void;
```

Comment:
Usually the `onFinished` callback has a `success` boolean parameter sent. Is that the case here too? 

It's sometimes a bit inconsistent throughout the platform and not all dialogs implement it

Reply:
There's an IDialogProps you can/should extend your iface on